specs/lib/gltfPipelineSpec.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var gltfPipeline = require('../../lib/gltfPipeline');
+var readGltf = require('../../lib/readGltf');
+var writeBinaryGltf = require('../../lib/writeBinaryGltf');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var gltfPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf';
+var glbPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.glb';
+
+describe('gltfPipeline', function() {
+    it('optimizes a gltf file, checks that a new gltf is defined when passing in a gltf file', function(done) {
```

Comment:
For this title, just `optimizes a gltf file` should be enough. Same comment below.